### ISSUE TITLE
Spam Detect and Punish!

### ISSUE THREAD

**Issue by SupaSickWog** [lapzbot/feature-requests#15]
So in simple form: Bot detects spam, Bot kicks user for spamming.... Possible? Maybe : It will help out the servers alot tho.

**Comment by lap00zza** [comment]
@lapzbot/teamlapzbot review this request.

**Comment by PaawanTono** [comment]
well that depends. Giving a warning 1st is nice. but can do that if it detects spam. but kicking and banning a user. i think that would depend on the server. there's a reason admins and moderators exists in most servers. Letting a bot do all of that is not a good idea according to me

**Comment by SeigaK** [comment]
I agree and disagree. While Pawan has a really good point; a "moderated mode" may be a useful feature.
For example, RH1-N0 Bot deletes long, repetitive strings it detects. Like "aaaaaaaa", "???????" and etc. 
So it may be a useful thing to make a feature where Lapz can have this enabled or disable by mods, say if there is many waves of spam caused by invasion or users being naughty.
Sorry for long post! xD
